hammerdb/Dockerfile

```diff
@@ -4,24 +4,28 @@ FROM centos/tools
 RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 RUN yum install -y expect python-elasticsearch wget tcl unixODBC-devel unixODBC
 
-# install packages required for ms-sql
 RUN curl https://packages.microsoft.com/config/rhel/7/prod.repo > /etc/yum.repos.d/mssql-release.repo
 RUN ACCEPT_EULA=Y yum -y install msodbcsql17
 RUN ACCEPT_EULA=Y yum -y install msodbcsql
 RUN yum remove -y unixODBC-utf16 unixODBC-utf16-devel && yum -y clean all
-RUN yum clean all -y
+RUN yum clean all -y 
 
-# clone the snafu repo
-#RUN git clone https://github.com/mkarg75/snafu.git
-# Download and install the hammer suite
+# clone the snafu repo 
+#RUN cd /opt/; git clone https://github.com/mkarg75/snafu.git
+RUN cd /opt/; git clone https://github.com/cloud-bulldozer/snafu.git
```

Comment:
since this dockerfile is inside snafu, we don't have to clone it but rather just move(copy) it and this can be done as follows:
```
RUN mkdir -p /opt/snafu/
COPY . /opt/snafu/
```